assets/css/main.css

```diff
@@ -14,3 +14,5 @@
 @import "sections/form";
 
 @import "pages/home";
+@import "tasks/global.css" ;
+@import "tasks/about-tasks.css" ;
```

Comment:
Do ścieżek w @import podajemy nazwę samego pliku. U Ciebie jest jeszcze dodane rozszerzenie ".css". Czyli zamiast @import "tasks/about-tasks.css", musi być @import "tasks/about-tasks". Dobrze, że pamiętałaś o usunięciu "_podłogi" :) nie wiem jak to się nazywa po polski.
Przed średnikami masz chyba spacje. Dobrze widzę? One są już nie porze.

Reply:
naprawione :)

Reply:
Każdy plik *.css ma się w nazwie rozpoczynać od _ ? tzn. ma być _global, _about-tasks, _blog-tasks, _contact-tasks, _article-tasks, prawda?

Reply:
Yes, exactly :)

Reply:
I have already deleted  low dash in global.css as you mentioned in comment to pull/7. Should I delete _ also in these PR files?

Reply:
Now, everything is great :) I see no more lodashes anywhere.